applications/nrf_desktop/configuration/nrf52833dk_nrf52820/child_image/mcuboot/mcuboot_private.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
quick question - was this generated for this board or is it a copy?

Reply:
It was generated.